src/components/SafariFooter.tsx

```diff
@@ -0,0 +1,16 @@
+/**
+ * Safari on iPhone prevents the user from scrolling to the bottom of the screen, due
+ * to the address bar.
+ *
+ * On Safari for iPhone, this React component adds an empty div with padding, which ensures that no content
+ * is hidden.
+ *
+ * On all other browsers, this has no effect.
+ */
+export const SafariFooter: React.FC = () => {
+  const isSafariOniPhone = navigator.userAgent.match(/(iPhone).*(Safari)/);
+
+  if (!isSafariOniPhone) return <></>;
```

Comment:
Shouldn't be a big deal, but this will also create a match for chrome on iOS : [Docs](https://developer.chrome.com/docs/multidevice/user-agent/#chrome-for-ios) .

Chrome UA:
```
Mozilla/5.0 (iPhone; CPU iPhone OS 10_3 like Mac OS X) 
AppleWebKit/602.1.50 (KHTML, like Gecko) CriOS/56.0.2924.75
Mobile/14E5239e Safari/602.1
```
Do you think we should also match on the string 'mobile'? Is iPad affected by this issue? If it is we may need to tweak the 'iPhone' string as well.

Reply:
I wasn't intending for this to affect Chrome, but I tested it and it suffers from the same problem.
![IMG_7526](https://user-images.githubusercontent.com/91760036/183574338-925b2dfa-dc9b-4e75-bb81-c7b64dc5ff2c.PNG)
 
I believe it's caused by the way we construct the header and scroll on main content. Probably a more significant fix, though.

Anyway, if this applies to Safari (or Chrome, etc) on iPhone, then we're good.

This regex shouldn't catch Safari on iPad, since it mentions "iPhone" specifically. I don't think the underlying issue should affect the iPad, given the screen size, but I'll test it in a few hours.

Reply:
Not an issue on iPad.
![IMG_0506](https://user-images.githubusercontent.com/91760036/183632066-46b0d5bc-7b92-49fe-b0dd-eff7ac4630f9.PNG)